src/components/DocumentModals/index.js

```diff
@@ -0,0 +1,11 @@
+import ShowDocumentLocalModal from './ShowDocumentLocalModal';
```

Comment:
Think maybe name of folder and `@namespace` might be better as ViewDocuments or something else, because there will be plenty of document modals used in `Forms`/`Documents`.  May be better for readability if this was something else.

Reply:
Thinking of just calling it modals and moving everything else in here in the future

Reply:
Have updated the directory to just Modals, thinking of having other modals moved into here in the future like LogoutModal, AddClientModal, etc.